### DISCUSSION TITLE
Agents not respecting the base url mentioned in env, and attempting to authenticate against default urls.

### DISCUSSION THREAD

**Discussion by baalajimaestro** [danny-avila/LibreChat#4870] [Troubleshooting]
![image](https://github.com/user-attachments/assets/581399ae-ade2-474e-8f2a-e28246cd50c3)

**Comment by yashsinha-aero** [comment]
+1.
I am using an openai key here with OPENAI_REVERSE_PROXY with the same issue

**Comment by danny-avila** [comment]
I'm acknowledging this, looking into it (specifically for `OPENAI_REVERSE_PROXY`)

**Reply by yashsinha-aero** [reply]
Hey there.
Are there any updates to this?
